### PR TITLE
LPS-170953 Create a field in the table and bring its value

### PR DESCRIPTION
**Pull request by caiosouza15**

### CODE REVIEW

**Review by samuelleolopes — APPROVED** [review]
Good job :fire:

**Review by rafaelfs98 — APPROVED** [review]
very very good :fire:

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/SubtaskCaseResults.tsx [hunk 1]
```diff
@@ -156,7 +156,13 @@ const SubtasksCaseResults = () => {
 						size: 'md',
 						value: i18n.translate('case'),
 					},
-					{key: 'issues', value: i18n.translate('issues')},
+					{key: 'issues',
+					render: (
+						_,
+						testraySubTaskCaseResult: TestraySubTaskCaseResult
+					) => testraySubTaskCaseResult.caseResult?.issues,
+
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/SubtaskCaseResults.tsx:164]: Space??

```suggestion
```

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/SubtaskCaseResults.tsx (hunk 1 above)
Comment (caiosouza15) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/SubtaskCaseResults.tsx:164]: Oooh sorry

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by caiosouza15** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**